### DISCUSSION TITLE
can't add new package

### DISCUSSION THREAD

**Discussion by marcgrin** [vuetifyjs/vuetify#17514] [Q&A]
在vue项目中add vuetify后，每次npm i 都会报错，似乎不让装新的包了？

**Comment by yuwu9145** [comment]
是否可以截图具体错误

**Reply by marcgrin** [reply]
![3WUF`3JK7$H$A__ GGB`5$A](https://github.com/vuetifyjs/vuetify/assets/107984308/09bfc097-4f53-4b35-9af7-4c8eb98c48a2)
百度和GPT查不出来怎么解决，或许是我的问题罢(悲)

**Reply by yuwu9145** [reply]
升级项目typescript 到 `^5.0.2`

**Reply by marcgrin** [reply]
> 升级项目typescript 到 `^5.0.2`

我的typescript版本是5.0.4，依旧是有这样的问题

**Reply by yuwu9145** [reply]
请提供一下 reproduce repo

**Reply by marcgrin** [reply]
> 请提供一下 reproduce repo

我似乎没有听说过这个东西，实在抱歉可能是我的学习还不够而不一定是vuetify出现的问题，我会继续学习尝试找到问题的来源，这里我提供一下我的vuecli版本@vue/cli 5.0.8以及ts版本Version 5.0.4
